eva_submission/eload_ingestion.py

```diff
@@ -77,13 +77,16 @@ def ingest(
             output_dir = self.run_accession_workflow(vcf_files_to_ingest)
             shutil.rmtree(output_dir)
             self.insert_browsable_files()
+            self.update_browsable_files_with_date()
+            self.update_files_with_ftp_path()
             self.refresh_study_browser()
 
         if do_variant_load:
             self.eload_cfg.set(self.config_section, 'variant_load', 'vep', 'version', value=vep_version)
             self.eload_cfg.set(self.config_section, 'variant_load', 'vep', 'cache_version', value=vep_cache_version)
             output_dir = self.run_variant_load_workflow(vep_version, vep_cache_version, skip_annotation, vcf_files_to_ingest)
             shutil.rmtree(output_dir)
+            self.update_loaded_assembly_in_browsable_files()
```

Comment:
Just thinking "out loud" on putting this step last - we could _maybe_ do some log parsing (like in the [accessioning workflow](https://github.com/EBIvariation/eva-submission/blob/master/eva_submission/nextflow/accession.nf#L158)) to check that the variant load step completed even if other steps in the load pipeline failed, since I assume that's the only prerequisite to this finalisation step.  Probably not for this PR but we can consider something like this for making the ingestion process a bit less painful while we work on making things more robust.

Reply:
That could be an idea.